.travis.yml

```diff
@@ -10,4 +10,4 @@ notifications:
   flowdock:
     secure: eqo+1HteeVzHGckSQhvSG18sD7Eq/DdFSouaW3Mj7owW8gH83/utYgCU2DOiSn404wOqIv2MgI+CYzgoFQCoYaR98BgxYS2iike1GWkn+s53VFSS9WNkjWW/4W7Fm+g4fWEDnN00U4mIJK3G/9l+C2zDiSaRwuD+9K61o3QCWaM=
 branches:
-  only: [master, staging, production]
+  only: [master, staging, production, pobrien/server_side_client_state]
```

Comment:
Yep, this is needed; otherwise, no travis builds for the PR.  And the name has to match exactly, or you get multiple travis builds otherwise.